src/pages/Quiz/QuizPage.jsx

```diff
@@ -0,0 +1,128 @@
+import { useState } from "react";
+import PageLayout from "../../components/PageLayout";
+import Button from "@mui/material/Button";
+import LinearProgress from '@mui/material/LinearProgress';
+import { v4 as uuidv4 } from "uuid";
+import { Box } from "@mui/system";
+import { useTheme } from '@mui/material/styles';
+
+/**
+ * Renders the Assessment page.
+ * @returns {JSX.Element} QuizPage component
+ */
+
+function QuizPage() {
+
+    const theme = useTheme();
+    const [currentQuestion, setCurrentQuestion] = useState(0);
+    const [progressValue, setProgressValue] = useState(1);
+
+    const handleAnswerButtonClick = (answer) => {
+        if (dummyQuestions[currentQuestion].correct === answer.target.textContent) // checks if the answer is correct
+            console.log("Correct");
+        else
+            console.log("Incorrect");
+
+        if (currentQuestion !== dummyQuestions.length - 1) { 
+            setCurrentQuestion(currentQuestion + 1);
+            setProgressValue((((currentQuestion +1)/ dummyQuestions.length) * 100));
+            return;
+        }
+
+        // update the progress bar on the last question
+        if (currentQuestion === dummyQuestions.length-1)
+            setProgressValue(100); 
+    }
+
+    const nextButtonClick = () => {
+        return;
+    }
+
+    const dummyQuestions = [ // placeholder for the real questions
```

Comment:
Remove that comment and please move this mockData to mockData.js

Reply:
It has been relocated. 
I also added a random question selection which is now incredibly secure >:)
If that needs to leave again, I would completely understand.